storage/rocksdb/rdb_threads.cc

```diff
@@ -61,17 +64,6 @@ int Rdb_thread::create_thread(const std::string &thread_name
   int err = mysql_thread_create(background_psi_thread_key, &m_handle, nullptr,
                                 thread_func, this);
 
-  if (!err) {
```

Comment:
my_thread* interface does not have facility to set thread name and IMHO this is bad form to mix use of pthread primitives with my_thread*. It is of little real use IMHO but I did make suggestion in commit msg that maybe we properly extend my_thread to expose thread naming functionality and then we can use that from here.